test/e2e/common/setup.go

```diff
@@ -66,6 +66,11 @@ const (
 	WestClusterName = "cluster-1"
 )
 
+type InstallOptions struct {
+	EnableSpire      bool
+	SetRemoteDNSName bool
+}
```

Comment:
I have something similar in my coming PR, so will adjust when working on massive rebase xD